### PR TITLE
Update last release's CHANGELOG with some tips

### PR DESCRIPTION
**Pull request by AliSoftware**
Note that we should also think about providing an error message when `swiftgen images` is passed a non-`xcassets` directory (instead of making the command line succeed without error but have the generated file contain "// No images found")

\cc @djbe
\ref #175  #214 (imports)
\ref #199 #207 (assets catalogs)
\ref #213 (multiple paths)

### CODE REVIEW

**Comment by djbe** [comment]
We'll have to do this based on the "extension" of the folder, because `actool` happily tries to parse a folder structure but does not then "dive" into catalog "files", and ignores any other files. Sample output from `xcrun actool --print-contents UnitTests/`:
```xml
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>com.apple.actool.catalog-contents</key>
	<array>
		<dict>
			<key>children</key>
			<array>
				<dict>
					<key>filename</key>
					<string>expected</string>
				</dict>
				<dict>
					<key>filename</key>
					<string>Filters</string>
				</dict>
				<dict>
					<key>children</key>
					<array>
						<dict>
							<key>filename</key>
							<string>Fonts</string>
						</dict>
						<dict>
							<key>filename</key>
							<string>Storyboards-iOS</string>
						</dict>
						<dict>
							<key>filename</key>
							<string>Storyboards-OSX</string>
						</dict>
					</array>
					<key>filename</key>
					<string>fixtures</string>
				</dict>
				<dict>
					<key>filename</key>
					<string>TestSuites</string>
				</dict>
			</array>
			<key>filename</key>
			<string>UnitTests</string>
		</dict>
	</array>
</dict>
</plist>
```

**Comment by AliSoftware** [comment]
Yeah, I was more thinking about checking each file extension before invoking `actool` on each of them

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,7 +38,7 @@
 * The `swiftgen images` command now uses the `actool` utility to parse asset catalogs,
   ensuring that the parser correctly handles namespaced folders.  
   ⚠️ Note that you now have to specify the exact path to your `.xcassets` assets catalogs
-  when using `swiftgen images` (and not just a directory to parse)
+  when using `swiftgen images` (and not just a directory to parse).
```
Comment (AliSoftware) [inline CHANGELOG.md]: Hahaha for once it's me who misformatted the CHANGELOG 😆

**Comment by djbe** [comment]
Jup, #217 should be merged alongside this.

**Comment by AliSoftware** [comment]
Cool!

CI might take some time, it seems like Travis is having issues again: https://www.traviscistatus.com/incidents/nq0sf0srvx8s

**Pull request merged**